generic/tests/pktgen_perf.py

```diff
@@ -67,12 +67,18 @@ def init_vm_and_login(test, params, env, result_file, pktgen_runner):
         process.system_output("numactl --hardware")
         process.system_output("numactl --show")
         _pin_vm_threads(params.get("numa_node"))
-        guest_ver = session_serial.cmd_output(guest_ver_cmd)
+        guest_ver = session_serial.cmd_output(guest_ver_cmd).strip()
         result_file.write("### guest-kernel-ver :%s" % guest_ver)
```

Comment:
This resulted in a single-line output
"### guest-kernel-ver :6.11.0-26.el10.x86_64Script:pktgen_perf Category:tx"

Reply:
Already corrected